packages/react-jss/src/withStyles.js

```diff
@@ -79,6 +77,61 @@ export default function withStyles<Theme: {}, S: Styles<Theme>>(
       // $FlowFixMe
       static defaultProps = {...InnerComponent.defaultProps}
 
+      static getSheetClasses(sheet, dynamicRules: ?DynamicRules) {
```

Comment:
actually we don't need this to be a static method, it can be a regular function outside of class

Reply:
Yeah, so for now I would like them to be `static`, once I start writing the hook I will most likely move them out and make most functions implementation independent for reuse most of the code.

Reply:
Why do you want them to be static?

Reply:
I think right now they are part of the component, but I want to abstract the whole creating/updating/removing of the sheets for the hook.